<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newConfGUI</class>
 <widget class="QWidget" name="newConfGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="title">
        <string>Â¿Que quieres?</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Seleccionar Individualmente cada carpeta</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Seleccionar carpeta raiz (El programa crea la estructura solo)</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>550</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <attribute name="title">
         <string>Individual</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Rutas:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Patterns:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pat">
           <property name="text">
            <string>Select Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_PatDir_2">
           <property name="text">
            <string>Actual dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_RefDir_2">
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Reference:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ref">
           <property name="text">
            <string>Select Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Actual dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_RoiDir_2">
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Rois</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_roi">
           <property name="text">
            <string>Select Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Actual dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_InputDir_2">
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Input Dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_input">
           <property name="text">
            <string>Select Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Actual dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_outputDir_2">
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Output Dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_output">
           <property name="text">
            <string>Select Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Actual dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_valDir_2">
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Validation Dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_val">
           <property name="text">
            <string>Select Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Actual dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Models Dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_selectModelDir">
           <property name="text">
            <string>Select Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>22</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>Raiz</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,3,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="text">
            <string>Carpeta raiz:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_root">
           <property name="text">
            <string>Select Dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Actual dir:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>DatasetBuilderConf (Ruta relativa):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>../cfg/datasetBuilderConfExample.cfg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Actual dir:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_saveNew">
       <property name="text">
        <string>Guardar Configuracion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_endNew">
       <property name="text">
        <string>Acabar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
